fs/rc/cache.go

```diff
@@ -41,3 +44,37 @@ func GetFsAndRemoteNamed(in Params, fsName, remoteName string) (f fs.Fs, remote
 func GetFsAndRemote(in Params) (f fs.Fs, remote string, err error) {
 	return GetFsAndRemoteNamed(in, "fs", "remote")
 }
+
+// GetExpireAndUnlink get the `expire` and `unlink` parameters from in
+func GetExpireAndUnlink(in Params) (expire time.Duration, unlink bool, err error) {
+	unlink, err = in.GetBool("unlink")
```

Comment:
I'd just call this in `rcPublicLink`

```go
unlink, _ = in.GetBool("unlink")
```

You don't want to give an error if the user doesn't pass the parameter you want it to default to `false`.

Likewise the `expire` parameter - missing should mean very large.